### PR TITLE
Preparation for publish

### PR DESCRIPTION
**Pull request by robertsipka**
Added features:
  - Restart function (only supported with JerryScript yet)
  - Automate debug server launch and tizen studio package installation
  - 'Create IoTjs Tizen Project' option to extension
  - Support delayed stack trace loading

### CODE REVIEW

**Review by tdusnoki — APPROVED** [review]
LGTM (informal)

**Review by LaszloLango — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 0.9.0
+ - Added features:
+   - Restart function (only supported with JerryScript yet)
+   - Automate debug server launch and tizen studio package installation
```
Comment (rtakacs) [inline CHANGELOG.md]: Tizen Studio

File: package.json [hunk 2]
```diff
@@ -88,16 +88,18 @@
...
-      "title": "Create IoT.js Tizen Project"
-    } 
-    ],
-    "menus": {
-      "editor/title": [ {
+    "commands": [
+      {
         "command": "iotjs-debug.createTizenProject",
-        "group": "navigation"
+        "title": "Create IoT.js Tizen Project"
```
Comment (rtakacs) [inline package.json:94]: `Project` -> `project`

Comment (robertsipka) [reply package.json:94]: This is a generated content

**Review by rtakacs — APPROVED** [review]
LGTM

**Pull request merged**